src/components/marquee-wizard/marquee-wizard.component.ts

```diff
@@ -27,6 +27,9 @@ export class MarqueeWizardComponent<TStepContext = any> extends WizardComponent
     /** If set to true the resizable splitter will be enabled and set to the default width **/
     @Input() resizable: boolean = false;
 
+    /** If set to false it will allow users to navigate to every step */
+    @Input() sequential: boolean = true;
```

Comment:
We shouldn't need to add this here is the wizard has it as it will inherit this input